Description: Squash compiler warning in ijs code
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=eb770ed
Author: Chris Liddell <chris.liddell@artifex.com>
Forwarded: yes
Last-Update: 2019-08-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ijs_exec_unix.c
+++ b/ijs_exec_unix.c
@@ -75,7 +75,7 @@
 
       argv[i++] = (char *)server_cmd;
       argv[i++] = NULL;
-      status = execvp (argv[0], argv);
+      status = execvp (argv[0], (char * const *)argv);
       if (status < 0)
 	exit (1);
     }
